modules/ROOT/pages/_partials/secrets-manager.adoc

```diff
@@ -0,0 +1,713 @@
+|`--cert-file` |Certificate file type | `--cert-file json`
+|`--expiration-date`| Expiration date for the secret. | `--expiration-date 01/01/2025`
+
+
+|===
+
+
+[[secret-keystore-create]]
+== secrets-mgr:keystore:create
+
+----
+> secrets-mgr:keystore:create [flags] 
+----
+
+Creates a new keystore secret in the secret group specified in `--group-id`, using the name specified in `--name` and the type specified in `--type`. 
+
+Besides the default flag `--help`, this command also accepts:
+
+[%header,cols="30a,40a,30a"]
+|===
+|Flag | Description |  Example
+|`--group-id`| Mandatory. Secret group ID. | `--group-id 030493`
+|`--name`|Mandatory. Name for your secret. | `--name TestSecret`
+|`--type` | Mandatory. Choose the keystore secret type. +
+Options: PEM, JKS, PKCS12, JCEKS. | `--type PEM`
+|`--algorithm` |Key manager factory algorithm for  JKS, PKCS12, and JCEKS keystore secrets. | `--algorithm PKIX`
+|`--alias`| Alias for the key used in JKS, PKCS12, and JCEKS keystore secrets. | `--alias KeyAliasExample`
+|`--capath-file`| CA path certificate file for PEM keystore secrets. | `--capath-file examplefile.pem`
+|`--expiration-date`| Expiration date for the secret| `--expiration-date 01/01/2025`
+|`--key-file`| Key file for PEM keystore secrets| `--key-file examplefile.pem`
+|`--key-passphrase`|Passphrase required for JKS, PKCS12 and JCEKS keystore secrets. Optional for PEM keystore secrets| `--key-passphrase examplePassphrase`
+|`--keystore-file` |Keystore filepath for JKS, PKCS12, and JCEKS type secrets. | `--keystore-file keystorefile`
+
+
+|===
+
+[[secret-keystore-describe]]
+== secrets-mgr:keystore:describe
+
+----
+> secrets-mgr:keystore:describe [flags] 
+----
+
+Returns the details of a keystore secret specified in `--id` from the secret group you pass in `--group-id`. 
+
+This command accepts the default flag `--help`.
+
+NOTE: The output does not include any sensitive or secret data.
+
+
+[[secret-keystore-list]]
+== secrets-mgr:keystore:list
+
+----
+> secrets-mgr:keystore:list [flags]
+----
+
+Lists all keystore secrets in a secret group specified in `--group-id`
+
+This command accepts the default flag `--help`.
+
+[[secret-keystore-modify]]
+== secrets-mgr:keystore:modify
+
+----
+> secrets-mgr:keystore:modify  [flags] 
+----
+
+Modifies the name or expiration date for a keystore secret specified in `--id` from the group you pass in`--groupid`. 
+
+Besides the default flag `--help`, this command also accepts:
+
+[%header,cols="30a,40a,30a"]
+|===
+|Flag | Description |  Example
+|`--name` | New name for the keystore secret | `--name TestKeystoreSecret`
+|`--expiration-date` | New expiration date for the keystore secret | `--expiration-date 01/01/2025`
+
+|===
+
+[[secrets-keystore-replace]]
+== secrets-mgr:keystore:replace
+
+----
+> secrets-mgr:keystore:replace  [flags] 
+----
+
+Replaces an existing keystore secret you pass in `--id`, from the secret group you pass in `--group-id`, using the type you pass in `--type`. 
```

Comment:
For consistency, it should be
Replaces an existing keystore secret **specified**  in `--id`...